constant.js

```diff
@@ -1,7 +1,8 @@
-const path = require('node:path');
+const PORT = 3000
+const MONGO_URL = 'mongodb+srv://admin:jcy1HCQ3hgVJn2q8@nodejshw.mnna64b.mongodb.net/db-contacts'
```

Comment:
PORT and MONGO_URL must be hidden in .env file. This file shouldn't be pushed in GitHub